File: Exercise/Easy/11. Check Divisibility/Code/Check Divisibility.py
def check_divisibility(num1, num2):
    if num1 % num2 == 0:
        return True
    return False

if __name__ == "__main__":
    num1, num2 = map(int, input().split())

    if check_divisibility(num1, num2):
        print("Divisible")
    else:
        print("Not Divisible")

File: Exercise/Easy/11. Check Divisibility/Code/Check Divisibility.cpp
#include <iostream>

using namespace std;

bool checkDivisibility(int num1, int num2) {
    if (num1 % num2 == 0) {
        return true;
    }
    return false;
}

int main() {
    int num1, num2;
    cin >> num1 >> num2;

    if (checkDivisibility(num1, num2)) {
        cout << "Divisible" << endl;
    }
    else {
        cout << "Not Divisible" << endl;
    }

    return 0;
}